std/range/package.d

```diff
@@ -3877,7 +3894,6 @@ pure unittest
     assert(b == [ "c", "b", "a" ]);
```

Comment:
> Perhaps sort!greaterThan would be easier to read, importing std.functional.
> edit: sorry, ignore this comment, that wouldn't work.

It does work, because numerics are before strings.
However I think that the key point of this example is more to show how to sort array b according to a. That's why I swapped "b" and "c", so that this becomes more obvious.